src/harbors/use-harbors.tsx

```diff
@@ -0,0 +1,76 @@
+import {useHarborsQuery} from '@atb/queries';
+import {useHarborConnectionOverrides} from '@atb/harbors/use-harbor-connection-overrides';
+import {useEffect, useState} from 'react';
+import {StopPlaceFragment} from '@atb/api/types/generated/fragments/stop-places';
+import {HarborConnectionOverrideType} from '@atb-as/config-specs';
+import {isDefined} from '@atb/utils/presence';
+import _ from 'lodash';
+
+export const useHarbors = (fromHarborId?: string) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [isSuccess, setIsSuccess] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [error, setError] = useState<unknown>();
+  const [data, setData] = useState<StopPlaceFragment[]>([]);
+
+  const harbors = useHarborsQuery();
```

Comment:
Why not just have all in our remote config instead of add overrides if something is missing? I harder to maintain to know what is missed on Entur and add those in another place.

MHO

Reply:
Well, that's the idea here. The connections defined in Firestore are added to the ones we receive from Entur

Reply:
I can expand a little on this. In general we have three options:
- **Always use overrides just from Firestore and not query Entur**
If a transport company (other than AtB) has a more "logical" boat-ticket setup, and they don't need tickets to span across different boat lines, then I think they would appriciate now having to hassle with adding the possible overrides to Firestore. For them Entur is good enough.

- **Use overrides in Firestore if they exist, if else use connections from Entur**
Could be done, and would maybe be more clear-cut logic. 

- **Use overrides in Firestore if they exist, and merge them with connections from Entur**
A little more complex logic, with the added benefit that if new harbors are added to Entur, it will still work even though somebody forgot to notify the development team.

I discussed this quickly with Kristian, and from my view I think both of the two last options are good, and don't really have a personal preference.

Reply:
It's the latter that's done in this PR on line 48 in `use-harbors.tsx`

Reply:
I will go for option 2, **Use overrides in Firestore if they exist only**, because it also will help in the boat free ticket which is not done by entur, they do not have that information yet, having it on our system and depends only on that for now should be fine. So that means that we need to add a new attribute here, to mark it as free between two stop places.

Reply:
I discussed this with @gorandalum, and we agreed to keep it as is, i.e. option 3 above. We believe that this will work well with free lines as well.